### PR TITLE
chore(examples): update event context

### LINKED ISSUE #8813: Locale example is not runnable on online playgrounds
### Environment

- Operating System: `Linux`
- Node Version:     `v16.14.2`
- Nuxt Version:     `3.0.0-rc.13`
- Nitro Version:    `0.6.1`
- Package Manager:  `npm@7.17.0`
- Builder:          `vite`
- User Config:      `modules`
- Runtime Modules:  `@nuxt/ui@0.3.3`
- Build Modules:    `-`

### Reproduction

- [Examples - Other - Locale](https://v3.nuxtjs.org/examples/other/locale)
- [CodeSandBox Playground](https://codesandbox.io/s/github/nuxt/framework/tree/main/examples/other/locale?hidenavigation=1&theme=light)
- [StackBlitz Playground](https://stackblitz.com/github/nuxt/framework/tree/main/examples/other/locale?file=app.vue&theme=light)

### Describe the bug

Cannot read properties of undefined (reading 'headers') because `ssrContext.req` should be `ssrContext.event.req`.

### Additional context

_No response_

### Logs

```shell
at eval (./composables/locale.ts:5:72)
at Module.useState (./node_modules/nuxt/dist/app/composables/state.mjs:21:26)
at useLocale (./composables/locale.ts:5:47)
at Module.useLocales (./composables/locale.ts:25:18)
at setup (./app.js:11:43)
at _sfc_main.setup (./app.js:65:23)
at callWithErrorHandling (./node_modules/@vue/runtime-core/dist/runtime-core.cjs.js:157:22)
at setupStatefulComponent (./node_modules/@vue/runtime-core/dist/runtime-core.cjs.js:7135:29)
at setupComponent (./node_modules/@vue/runtime-core/dist/runtime-core.cjs.js:7090:11)
```

### PR DESCRIPTION
**Pull request by cuebit**
### 🔗 Linked issue

#8813

### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

resolves #8813

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by danielroe — APPROVED** [review]

File: examples/other/locale/composables/locale.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ export const useDefaultLocale = (fallback = 'en-US') => {
   if (process.server) {
     // Learn more about the nuxtApp interface on https://v3.nuxtjs.org/docs/usage/nuxt-app#nuxtapp-interface-advanced
     const nuxtApp = useNuxtApp()
-    const reqLocale = nuxtApp.ssrContext?.req.headers['accept-language']?.split(',')[0]
+    const reqLocale = nuxtApp.ssrContext?.event.req.headers['accept-language']?.split(',')[0]
```
[the remark this answers is not in the document]
Comment (cuebit) [reply examples/other/locale/composables/locale.ts:10]: Even better catch! Nice.

[the remark this answers is not in the document]
Comment (cuebit) [reply examples/other/locale/composables/locale.ts]: Updated 👍

[the remark this answers is not in the document]
Comment (danielroe) [reply examples/other/locale/composables/locale.ts]: No problem! you're the one who spotted this 😊

File: examples/other/locale/composables/locale.ts [hunk 2]
```diff
@@ -6,8 +6,7 @@ export const useDefaultLocale = (fallback = 'en-US') => {
   const locale = ref(fallback)
   if (process.server) {
     // Learn more about the nuxtApp interface on https://v3.nuxtjs.org/docs/usage/nuxt-app#nuxtapp-interface-advanced
-    const nuxtApp = useNuxtApp()
-    const reqLocale = nuxtApp.ssrContext?.req.headers['accept-language']?.split(',')[0]
+    const reqLocale = useRequestHeaders()['accept-language']?.split(',')[0];
```
Comment (danielroe) [inline examples/other/locale/composables/locale.ts:9]:
```suggestion
    const reqLocale = useRequestHeaders()['accept-language']?.split(',')[0]
```

**Pull request merged**